resources/templates/_footer.html

```diff
@@ -77,10 +80,10 @@ <h3 style="color:#FFF;">Member Resources</h3>
                         <li>
                             <a href="https://constitution.csh.rit.edu/bylaws.pdf" class="sitemap-link">By-Laws</a></li>
                         <li>
-                            <a href="https://www.facebook.com/groups/ComputerScienceHouse/" class="sitemap-link"><i class="fa fa-facebook"></i> CSH on Facebook</a>
+                            <a href="https://www.facebook.com/groups/16764593277/" class="sitemap-link"><i class="fa fa-facebook"></i> CSH Facebook Group</a>
```

Comment:
Hmm alright. I mean we haven't had a public page until a month ago and I know for a fact that link was there when I first visited the site a year ago. It's under member resources so I expect it was meant to be the group link